src/RamTransforms.h

```diff
@@ -439,7 +439,7 @@ class TupleIdTransformer : public RamTransformer {
      * @param RAM program
      * @result A flag indicating whether the RAM program has been changed.
      *
-     * Search for RamSearches and RamElementAccesses and rewrite their tupleIds
+     * Search for RamTupleOperationes and RamTupleElementes and rewrite their tupleIds
```

Comment:
Search and replace has left you with too many 'e's in your plurals, and on line 415